app/serializers/subscriber_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# This handles basic serialization for Subscribers
+class SubscriberSerializer < ActiveModel::Serializer
+  attributes :id
```

Comment:
just do `attributes :id, :name, :metadata` instead of calling `attributes` over and over